### DISCUSSION TITLE
Check if the Cloudflare DoT works

### DISCUSSION THREAD

**Discussion by unclamped** [pluim003/docker-pihole-unbound#18] [Q&A]
Hey, is there a reliable way of checking if the Cloudflare forward zone works? I tried going to https://1.1.1.1/help, but it seems like I'm not connected to CF at all. Am I doing this wrong? :thinking:

**Comment by pluim003** [comment]
I don't know. What is the output? Can you show a screenshot?

Mine looks like this (after I added the DoT-stuff):

![afbeelding](https://user-images.githubusercontent.com/67138922/220042530-54506c80-8bff-4f4f-ad2e-01e8ea282aa7.png)

**Reply by pluim003** [reply]
Hmm... might be that my config has been messed up after I was trying to start up the original pihole-container. Because DoT is now not enabled.

**Reply by pluim003** [reply]
A bit later, after adding it again to the conf:

![afbeelding](https://user-images.githubusercontent.com/67138922/220044107-f123cf3b-591b-4180-9159-034eca42f25a.png)

But I also noticed that every now and then when going to this link it might say that I'm not connected to 1.1.1.1.
